geoips/commandline/log_setup.py

```diff
@@ -16,6 +16,24 @@
 import sys
 
 
+def log_with_emphasis(LOG, messages):
+    """Log the given message with emphasis provided a certain log type.
+
+    Parameters
+    ----------
+    LOG: logging function
+        Can be of any type [debug, info, interactive...]
+    message: str
```

Comment:
```python
>>> import geoips.commandline.log_setup as log_setup
>>> import logging
>>> LOG = logging.getLogger("fake_name")
>>> log_setup.log_with_emphasis(LOG.critical, ["A very long string just for testing this. "*5])
```

Outputs
```
    ********************************************************************************
    A very long string just for testing this. A very long string just for testing this. A very long string just for testing this. A very long string just for testing this. A very long string just for testing this. 
    ********************************************************************************
```

Reply:
but 
```python
>>> log_setup.log_with_emphasis(LOG.critical, "test")
```

Outputs
```
    *
    t
    e
    s
    t
    *
```

Reply:
Is this intended? Am I testing this correctly?

Reply:
If I *am* testing correctly, please fix. Else please update docstring with an example to help me/future folks out 🙂

Reply:
I can see a couple of different ways that this could work. This could accept a single string and word-wrap it, regardless of its contents. It could also accept multiple strings and do something intelligent with them like treat them as paragraphs to be wrapped separately. I'd lean towards the simple option just for KISS purposes unless there is a reason we want paragraphs.

With single strings, @biosafetylvl5's first example would just be this:
```
>>> import geoips.commandline.log_setup as log_setup
>>> import logging
>>> LOG = logging.getLogger("fake_name")
>>> log_setup.log_with_emphasis(LOG.critical, " ".join(["A very long string just for testing this. "*5]))
```

and their `"test"` example would work as-is:

```
>>> log_setup.log_with_emphasis(LOG.critical, "test")
*********
** test **
*********
```

Alternatively, if we allowed multiple messages in a single call, it could consider them to be paragraphs:

```
def log_with_emphasis(LOG, *messages):
    ...
```

And the call would look like this:
```
log_with_emphasis(LOG.critical, "A very long string just for testing this.", "A very long string just for testing this.", ...)
```

Which would make something like this:
```
***********************************************
** A very long string just for testing this. **
**                                           **
** A very long string just for testing this. **
**                                           **
** A very long string just for testing this. **
***********************************************
```

Reply:
Oh, I was expecting this to be a list of strings, with a newline between each string, and the ***s before and after the lines.

We could just check if it is a list in the log_with_emphasis function, and print a single line if not, else loop through the strings in the list if it is a list ?

Reply:
I like what you're thinking @mindyls! I think you're right about KISS, so maybe we should just word-wrap single strings? If you have a list of strings you can always call the log function multiple times.

eg.

```python
for string in ["A very long string just for testing this.", "A very long string just for testing this.", ...]:
    log_with_emphasis(LOG.critical, string)
```

Output:
```
***********************************************
** A very long string just for testing this. **
***********************************************
***********************************************
** A very long string just for testing this. **
***********************************************
***********************************************
***********************************************
** A very long string just for testing this. **
***********************************************
... (etc.)
```

Reply:
I would rather just have it replicate what it did before

* pass in a list of strings
* Put "*"*80 before the strings
* print the strings one at a time
* Put "*"*80 after the strings.

If we wanted to allow passing an individual string as well (to avoid your 
t
e
s
t
above), we could just check in log_with_emphasis if a list or string was passed in.